### ISSUE TITLE
packaging for Mac should not put 2 cases of README in case-insensitive filesystem

### ISSUE THREAD

**Issue by sudoer** [sedwards2009/extraterm#130]
When trying to copy Extraterm.app from the packaged DMG file to my Mac, I get an error because there are two node modules that includes two README files with different capitalizations, and the Mac filesystem (by default) is case-insensitive.

```
ls -l Extraterm.app/Contents/Resources/app/node_modules/colors/R* Extraterm.app/Contents/Resources/app/node_modules/ms/{READ,read}*.md
-rw-r--r--  1 alanporter  staff  3980 Oct 19 16:02 Extraterm.app/Contents/Resources/app/node_modules/colors/README.md
-rw-r--r--  1 alanporter  staff  3190 Oct 19 16:02 Extraterm.app/Contents/Resources/app/node_modules/colors/ReadMe.md
-rw-r--r--  1 alanporter  staff   933 Oct 19 16:02 Extraterm.app/Contents/Resources/app/node_modules/ms/README.md
-rw-r--r--  1 alanporter  staff  1721 Oct 19 16:02 Extraterm.app/Contents/Resources/app/node_modules/ms/readme.md
```

Dragging the file to the Applications folder causes this error:

You can’t copy “Extraterm.app” because it has the same name as another item on the destination volume, and that volume doesn’t distinguish between upper- and lowercase letters in filenames.

Work-around: use rsync to copy the Extraterm.app folder into /Applications.  It will simply overwrite the other readme files when it encounters the conflict.

**Comment by sedwards2009** [comment]
thanks for reporting this. I'm guessing you are using a different file system or different settings than the mac I tested on. Do you know if the zip for mac works?

**Comment by sudoer** [comment]
The zip file produces a similar warning when it gets to the README files.

```
replace extraterm-0.37.0-darwin-x64/Extraterm.app/Contents/Resources/app/node_modules/colors/README.md? [y]es, [n]o, [A]ll, [N]one, [r]ename:
```

I am running the default case-insensitive filesystem.

**Comment by sedwards2009** [comment]
Fix has been released in 0.37.1.
